modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java

```diff
@@ -2842,8 +2842,15 @@ else if (valueString.isEmpty()) {
 			}
 		}
 		else if (sqlType == Types.DECIMAL) {
+			String decimalValue = String.valueOf(value);
+
+			if (decimalValue.contains(StringPool.COMMA)) {
+				StringUtil.replace(
+					String.valueOf(value), CharPool.COMMA, CharPool.PERIOD);
```

Comment:
In this method StringUtil.replace(), shouldn't the first parameter be the variable "decimalValue" instead of "String.valueOf(value)"?

Reply:
I faced an issue on my local ENV, sorry about my mistake! Thank you